app-i18n/fbterm/fbterm-1.7_p20190503.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit autotools fcaps toolchain-funcs
+
+DESCRIPTION="Fast terminal emulator for the Linux framebuffer"
+HOMEPAGE="https://github.com/gjedeer/fbterm"
+SRC_URI="https://github.com/gjedeer/fbterm/archive/ccea326dd73f4d6b6442fde7ba7c2be9cd35c6df.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/fbterm-ccea326dd73f4d6b6442fde7ba7c2be9cd35c6df"
```

Comment:
Maybe you'll want to use a `MY_COMMIT` variable to make future bumps easier.
Kinda like what I did here:
https://github.com/gentoo/gentoo/blob/4f620a3ba9edc210e2a55eff7c5c71efd713f4ef/net-analyzer/nagios-plugin-check_lm_sensors/nagios-plugin-check_lm_sensors-4.1.1_p20191028.ebuild#L9